13-Roman_Data_Models/13-Roman_Data_Models.ipynb

```diff
@@ -0,0 +1,624 @@
+{
```

Comment:
<p>Install the latest gwcs in your env to remove the warnings.</p><p> </p><p>And change the name of the <code>cal</code> file </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>